languagetool-language-modules/pt/src/main/java/org/languagetool/rules/pt/PortugueseProclisisFilter.java

```diff
@@ -23,8 +21,10 @@ public RuleMatch acceptRuleMatch(RuleMatch match, Map<String, String> arguments,
     HashSet<String> suggestions = new HashSet<>(Collections.emptyList());
     for (AnalyzedToken at : encliticVerbTokenReadings.getReadings()) {
       String posTag = at.getPOSTag();
+      if (posTag == null || (!posTag.startsWith("V") && !posTag.contains(":"))) {
```

Comment:
Maybe OR instead of AND here to require both conditions?
>  if (posTag == null || !posTag.startsWith("V") || !posTag.contains(":")) {

Reply:
Agreed, done at 42e0f0bd6f1